webapi/src/main/scala/org/knora/webapi/responders/v2/ontology/Cache.scala

```diff
@@ -201,9 +201,10 @@ object Cache extends LazyLogging {
       exception match {
         case inconsistentRepositoryDataException: InconsistentRepositoryDataException =>
           log.error(inconsistentRepositoryDataException.message)
-          SuccessResponseV2(
-            s"An error occurred when loading ontologies: ${inconsistentRepositoryDataException.message}"
-          )
+          // SuccessResponseV2(
+          //   s"An error occurred when loading ontologies: ${inconsistentRepositoryDataException.message}"
+          // )
+          throw inconsistentRepositoryDataException
```

Comment:
I'm tempted to say we don't even need to log this here (the exception would propagate out anyways), so we could just not touch the future and return it right away. What do you think?

Reply:
Yes, I agree, either log or re-throw an exceptions, but never do both. Otherwise the exception ends up getting logged multiple times in multiple layers.